src/Components/Users/UserModifierForm.js

```diff
@@ -0,0 +1,143 @@
+        };
+        if(!isNotValidField('profilePhoto') && !isValidImageType('type') ){
+            errors.profilePhoto = 'Tipo de archivo no soportado, extensiones permitidas: jpg o png'
+        }
+
+        return errors;
+    };
+
+    const {values,touched,errors,handleSubmit,handleChange,handleBlur,setFieldValue} = useFormik({
+        initialValues: {
+            name: user?.name ?? '',
+            password: user?.password ?? '',
+            email: user?.email ?? '',
+            roleId:user?.roleId ?? '',
+            profilePhoto:user?.profilePhoto ?? {}
+        },
+        validate,
+        enableReinitialize:true,
+        onSubmit: values => handleOnSubmit(values)
+    });
+
+    return (
+        <form className="form-container" onSubmit={handleSubmit}>
+            {!hasUser              
+                ?<div>
+                    <img alt="profilePhoto" src={values.profilePhoto}></img>
+                </div>
+                :<div>
+                    <label htmlFor="img">Seleccionar imagen: </label>
+                    <input
+                        type="file"
+                        id="profilePhoto"
+                        accept="image/jpeg"
+                        onChange={e=>setFieldValue('profilePhoto',e?.target?.files ? e.target.files[0] : null)}
+                        onBlur={handleBlur}
+                    />
+                    {touched.profilePhoto && errors.profilePhoto}
+                </div>
+            }
+            <input 
+                className="input-field" 
+                type="text" 
+                id="name" 
+                value={values.name} 
+                onChange={handleChange} 
+                onBlur={handleBlur}
+                placeholder="Nombre"
+            />
+            {touched.name && errors.name}
+            <input 
+                className="input-field" 
+                type="text" 
+                id="email" 
+                value={values.email} 
+                onChange={handleChange}
+                onBlur={handleBlur} 
+                placeholder="Email"
+            />
+            {touched.email && errors.email}
+            <input 
+                className="input-field" 
+                type="password" 
+                id="password" 
+                value={values.password} 
+                onChange={handleChange}
+                onBlur={handleBlur} 
+                placeholder="Contraseña"
+            />
+            {touched.password && errors.password}
+            <select 
+                className="input-field"
+                value={values.roleId}
+                id="roleId"
+                onChange={handleChange}
+            >
+                <option value="" disabled >Seleccione el rol</option>
+                {
+                    rolesList.map(role => {
+                        return <option key={role.roleId} value={role.roleId}>{role.roleId}</option>
```

Comment:
this return is not necessary if you delete the brackets. you can use ()

Reply:
Perfect! I already change it